src/components/Footer.jsx

```diff
@@ -1,10 +1,122 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Box,
+  Container,
+  Typography,
+  Grid,
+  Link as MuiLink,
+} from '@mui/material';
+import {
+  genMailToLink,
+  reportIssue,
+  featureRequest,
+  withDeviceInfo,
+} from '../lib/mail';
+import { SUBSCRIPTION_FORM_URL } from '../config/constants';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+const quickLinks = {
+  'footer.addScholarship': '/scholarships/new',
+  'footer.browseScholarship': '/scholarships',
+};
+
+const companyLinks = {
+  'footer.about': '/about',
+  'footer.contact': '/contact',
+};
+
+const helpLinks = {
+  'footer.reportIssue': genMailToLink({
+    subject: 'Bug Report',
+    body: withDeviceInfo(reportIssue),
+  }),
+  'footer.suggestIdea': genMailToLink({
+    subject: 'Feature Request',
+    body: withDeviceInfo(featureRequest),
+  }),
+  'footer.subscribeForUpdates': SUBSCRIPTION_FORM_URL,
+  'footer.reachOut': genMailToLink({
+    subject: 'Outreach',
+    body: 'Please describe the purpose of your outreach below.\n',
+  }),
+};
+
+function FooterColumn({ title, links, internal, t }) {
+  return (
+    <Grid item>
+      <Typography
+        color={(theme) => theme.palette.grey[500]}
+        sx={{ fontWeight: 'bold' }}>
+        {t(title)}
+      </Typography>
+      {Object.entries(links).map(([name, link]) => (
+        <Box key={name}>
+          <MuiLink
+            component={internal ? Link : MuiLink}
+            to={internal ? link : ''}
+            href={!internal ? link : ''}
+            variant="subtitle2"
+            color="text.secondary"
+            underline="hover">
+            {t(name)}
+          </MuiLink>
+        </Box>
+      ))}
+    </Grid>
+  );
+}
+
+FooterColumn.propTypes = {
+  title: PropTypes.string.isRequired,
+  links: PropTypes.objectOf(PropTypes.string).isRequired,
+  internal: PropTypes.bool,
+  t: PropTypes.func,
+};
+FooterColumn.defaultProps = {
+  internal: false,
+};
+
+function Footer(props, ref) {
+  const { t } = useTranslation();
 
-function Footer() {
   return (
-    <div>
-      <p />
-    </div>
+    <Box sx={{ bgcolor: 'background.secondary' }} ref={ref}>
+      <Container maxWidth="lg" sx={{ py: 4 }}>
```

Comment:
```suggestion
      <Container maxWidth="lg">
```

I think the Grid already introduces a padding of 3

Reply:
Nop :(
<img width="1240" alt="Screen Shot 2022-03-06 at 9 00 53 PM" src="https://user-images.githubusercontent.com/40483569/156970934-a3a51605-2b24-448f-9c10-eb67367b055b.png">